src/peft/tuners/lokr/layer.py

```diff
@@ -156,10 +156,11 @@ def update_layer(
             raise ValueError(f"`r` should be a positive integer value but the value passed is {r}")
 
         self.r[adapter_name] = r
-        self.alpha[adapter_name] = alpha
-        self.scaling[adapter_name] = alpha / r
+        self.alpha[adapter_name] = alpha or r
```

Comment:
Could you explain why this is needed?

Reply:
It is used in case when alpha is 0. So we set to `r` hence no scaling would be performed

Reply:
Hmm, not sure about this one. Could the user not set `alpha = r` directly? Or is there some precedence that 0 implies that `scaling=1`?